hackathon_site/event/views.py

```diff
@@ -203,3 +205,13 @@ class TeamListView(mixins.ListModelMixin, generics.GenericAPIView):
 
     def get(self, request, *args, **kwargs):
         return self.list(request, *args, **kwargs)
+
+
+class TeamIncidentDetailView(mixins.RetrieveModelMixin, generics.GenericAPIView):
+    queryset = Incident.objects.all()
+
+    serializer_class = IncidentListSerializer
+    permission_classes = [FullDjangoModelPermissions]
```

Comment:
Since this is a participant, you should check that they have logged in with a profile, since participants won't be given view permissions like this. 

You can use UserHasProfile permission class (the other views in api_views.py do the same)